media-libs/tiff/files/tiff-4.0.10-CVE-2019-17546-tif_getimage-overflow.patch

```diff
@@ -0,0 +1,89 @@
+diff --git a/libtiff/tif_getimage.c b/libtiff/tif_getimage.c
```

Comment:
you'll want to add metadata to the patch header
https://dev.gentoo.org/~vapier/clean-patches

Reply:
Done and Done.